Source/WebCore/page/Quirks.cpp

```diff
@@ -892,6 +893,12 @@ bool Quirks::shouldBypassBackForwardCache() const
             return documentLoader->response().cacheControlContainsNoStore();
     }
 
+    // Spinner issue from image search for bing.com.
+    if (registrableDomain == "bing.com"_s) {
+        static MainThreadNeverDestroyed<const AtomString> imageSearchDialogID("sb_sbidialog"_s);
```

Comment:
When I go to bing.com and run `document.getElementById("sb_sbidialog")` then it returns an element, yet the image search dialog is not showing. Therefore this quirk doesn't seem specific enough.

Reply:
I can detect whether the pane is visible or not via this in JS:
```
window.getComputedStyle(document.getElementById("sb_sbidialog")).display
< "none"
< "block"
```

We need to translate that into C++

Reply:
Alan suggests checking that `element->renderer()` to make sure it's visible.

Reply:
implemented.